mne/io/tests/test_raw.py

```diff
@@ -87,8 +87,12 @@ def _test_raw_reader(reader, test_preloading=True, **kwargs):
     assert_equal(concat_raw.first_samp, first_samp)
     assert_equal(concat_raw.last_samp - last_samp + first_samp, last_samp + 1)
     idx = np.where(concat_raw.annotations.description == 'BAD boundary')[0]
-    assert_array_almost_equal([(last_samp - first_samp) / raw.info['sfreq']],
-                              concat_raw.annotations.onset[idx], decimal=2)
+
+    expected_bad_boundary_onset = last_samp / raw.info['sfreq'] if concat_raw.info['meas_date'] is not None else (last_samp - first_samp) / raw.info['sfreq']
```

Comment:
PEP8

```py
if concat_raw.info['meas_date'] is None:
	expected_bad_boundary_onset = (last_samp - first_samp) / raw.info['sfreq']
else:
	expected_bad_boundary_onset = last_samp / raw.info['sfreq']
```

Reply:
you're editor should prevent you from saving a non-pep8 file ! :)